### PR TITLE
Adds basic tests for routing_handlers

### PR DESCRIPTION
**Pull request by saurav-c**

### CODE REVIEW

File: tests/bedrock/route/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#  limitations under the License.
+
+CMAKE_MINIMUM_REQUIRED(VERSION 3.6)
+
+SET(KVS_SRC_DIR ${CMAKE_SOURCE_DIR}/src/bedrock/kvs)
+
+FILE(GLOB HANDLER_SOURCES "${KVS_SRC_DIR}/*handler.cpp")
+FILE(GLOB MOCK_SOURCES "${CMAKE_SOURCE_DIR}/tests/mock/*.cpp")
+
+ADD_EXECUTABLE(run_server_handler_test
```
Comment (cw75) [inline tests/bedrock/route/CMakeLists.txt]: should be `run_router_handler_test`?

File: tests/bedrock/route/routing_handler_base.hpp [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    spdlog::basic_logger_mt("mock_logger", "mock_log.txt", true);
+
+class RoutingHandlerTest : public ::testing::Test {
+ protected:
+  Address ip = "127.0.0.1";
+  unsigned thread_id = 0;
+  std::unordered_map<unsigned, GlobalHashRing> global_hash_ring_map;
+  std::unordered_map<unsigned, LocalHashRing> local_hash_ring_map;
+  std::unordered_map<Key, KeyInfo> placement;
+  std::unordered_map<Key, unsigned> key_size_map;
```
Comment (cw75) [inline tests/bedrock/route/routing_handler_base.hpp]: router does not use `key_size_map`, `ServerThread`, `Serializer`, and `MemoryKVS`

**Comment by vsreekanti** [comment]
Looks like the build failed because the code wasn't formatted correctly, @saurav-c. Run `make clang-format` from the build directory, and that should fix it.

**Comment by vsreekanti** [comment]
`clang-format` is still finding changes to files, @saurav-c.

**Comment by cw75** [comment]
and seems like we have a missing ";" in `membership_handler`?

**Pull request closed without merging**